lib/actions/ui.js

```diff
@@ -121,36 +123,36 @@ export function moveRight () {
   };
 }
 
-export function moveTo (i) {
+export function moveTo(i) {
   return (dispatch, getState) => {
     dispatch({
       type: UI_MOVE_TO,
       index: i,
-      effect () {
-        const { sessions } = getState();
+      effect() {
+        const {sessions} = getState();
         const uid = sessions.activeUid;
         const sessionUids = keys(sessions.sessions);
         if (uid === sessionUids[i]) {
           console.log('ignoring same uid');
-        } else if (null != sessionUids[i]) {
```

Comment:
Generally I agree, however I'm a bit unsure when it comes to config values. If it never makes sense for a specific config value to be null it might be better to prevent plugins and similar from setting it as such.